jenkins/scripts/coverage/README.md

```diff
@@ -107,8 +108,5 @@ Host coveragedata
 The results are pushed to /home/benchmark/coverage-out/out.
 
 # Transfer to the website
-As mentioned earlier, the website will pull updates hourly from
-/home/benchmark/coverage-out/out and put
-them in the right place to be served at coverage.nodejs.org.  The key
-required to do this is already in place in order to support the similar process
-for benchmarking.nodejs.org
+
+The benchmark.nodejs.org website is archived and the data is not synced anymore.
```

Comment:
> I'd personally like to ditch https://coverage.nodejs.org/ and move to [codecov.io](https://app.codecov.io/gh/nodejs/node) (https://github.com/nodejs/build/issues/2797) as it will be one less thing for https://github.com/orgs/nodejs/teams/build-infra to maintain. cc @Trott

Yeah, I think that build should narrow their scope as much as is reasonable, and removing the coverage site that duplicates (or at least substantially overlaps with) the codecov.io site seems like the right thing to do, even if I would prefer otherwise.

Reply:
I'm +1 on removing the coverage site if codecov.io fills the requirement now.